### PR TITLE
Moving new Express QS out of Beta

### PR DESCRIPTION
**Pull request by davidpatrick**
Moving the new express-openid-connect tutorial out of beta and replacing our passortjs guidance with it

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /articles/quickstart @auth0/dx-sdks-approver
 /articles/libraries @auth0/dx-sdks-approver
 /articles/cms @auth0/dx-sdks-approver
-/articles/quickstart/webapp/express @pdillon
+/articles/quickstart/webapp/nodejs @pdillon
+/articles/quickstart/webapp/nodejs @davidpatrick
```
Comment (stevehobbsdev) [inline .github/CODEOWNERS]: Can these live on the same line?

```suggestion
/articles/quickstart/webapp/nodejs @pdillon @davidpatrick
```

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 2]
```diff
@@ -12,365 +12,103 @@ topics:
...
+<!-- markdownlint-disable MD002 MD041 -->
 
-<%= include('../_includes/_getting_started', { library: 'Node.js', callback: 'http://localhost:3000/callback' }) %>
+## Configure Auth0
 
-<%= include('../../../_includes/_logout_url', { returnTo: 'http://localhost:3000' }) %>
+You will need to register your application with Auth0 in order to start authenticating users. Go to the [Applications](${manage_url}/#/applications) screen in the Auth0 dashboard, create a new **Regular Web Application**, and follow the steps below.
 
-## Configure Node.js to use Auth0
+### 1. Configure Callback URL
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/nodejs/01-login.md]: Sub-heads [should use sentence case](https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#formatting):

```suggestion
### 1. Configure callback URL
```

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 3]
```diff
@@ -12,365 +12,103 @@ topics:
...
-Create the `.env` file in the root of your app and add your Auth0 variables and values to it.
+The callback URL for the application created in this quickstart is `http://localhost:3000/callback`. Paste that in the **Allowed Callback URLs** field for the application you just created.
 
-```
-# .env
-AUTH0_CLIENT_ID=${account.clientId}
-AUTH0_DOMAIN=${account.namespace}
-AUTH0_CLIENT_SECRET=YOUR_CLIENT_SECRET
-```
+### 2. Configure Logout URL
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/nodejs/01-login.md]: Sub-heads should use sentence case:

```suggestion
### 2. Configure logout URL
```

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 4]
```diff
@@ -12,365 +12,103 @@ topics:
...
+A logout URL is an application route that Auth0 can return users to after logging out. This URL must be registered with Auth0 or else users will be unable to log out of the application and will get a "misconfiguration" error.
 
-If you are using git, create a `.gitignore` file (or edit your existing one, if you have one already) and add `.env` to it. The `.gitignore` file tells source control to ignore the files (or file patterns) you list. Be careful to add `.env` to your `.gitignore` file and commit that change before you add your `.env`.
+The logout URL for the application created in this quickstart is `http://localhost:3000`. Paste that in the **Allowed Logout URLs** field for the application you just created, then scroll down and click **Save Changes**.
 
-```
-# .gitignore
-.env
-```
+### 3. Get Your Application Keys
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/nodejs/01-login.md]: Sub-heads should use sentence case:

```suggestion
### 3. Get your application keys
```

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 5]
```diff
@@ -12,365 +12,103 @@ topics:
...
-To get started, install the following dependencies.
+* **Domain**
+* **Client ID**
 
-* [passport](http://www.passportjs.org/) - an authentication middleware for Node.js
-* [passport-auth0](https://github.com/auth0/passport-auth0) - an Auth0 authentication strategy for Passport
-* [express-session](https://www.npmjs.com/package/express-session) - a middleware to manage sessions
-* [dotenv](https://www.npmjs.com/package/dotenv) - a module to load environment variables from a `.env` file
+## Integrate Auth0
+### 4. Install Dependencies
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/nodejs/01-login.md]: Sub-heads should use sentence case:

```suggestion
### 4. Install dependencies
```

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 6]
```diff
@@ -12,365 +12,103 @@ topics:
...
+Your application will need the [`express-openid-connect`](https://github.com/auth0/express-openid-connect) package which is an Auth0-maintained OIDC-compliant library for Express.
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/nodejs/01-login.md]: Tweak for conciseness:

```suggestion
Your application will need the [`express-openid-connect`](https://github.com/auth0/express-openid-connect) package, an Auth0-maintained OIDC-compliant library for Express.
```

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 7]
```diff
@@ -12,365 +12,103 @@ topics:
...
-  secret: 'CHANGE THIS TO A RANDOM SECRET',
-  cookie: {},
-  resave: false,
-  saveUninitialized: true
-};
-
-if (app.get('env') === 'production') {
-  // Use secure cookies in production (requires SSL/TLS)
-  sess.cookie.secure = true;
+### 5. Configure Router
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/nodejs/01-login.md]: Sub-heads should use sentence case:

```suggestion
### 5. Configure router
```

**Review by stevehobbsdev — COMMENTED** [review]
Looks good! Just some minor suggestions.

File: articles/quickstart/webapp/nodejs/01-login.md [hunk 8]
```diff
@@ -12,365 +12,103 @@ topics:
...
+### 5. Configure router
+The Express OpenID Connect library provides the `auth` router in order to attach authentication routes to your application. You will need to configure the router with the following configuration keys:
 
-  // Uncomment the line below if your application is behind a proxy (like on Heroku)
-  // or if you're encountering the error message:
-  // "Unable to verify authorization request state"
-  // app.set('trust proxy', 1);
-}
+- `baseURL` - The URL where the application is served
+- `appSessionSecret` - A long, random string
```
Comment (pdillon) [inline articles/quickstart/webapp/nodejs/01-login.md]: Should this be now renamed to `appSession.secret` or something else to reflect the change?

Comment (davidpatrick) [inline articles/quickstart/webapp/nodejs/01-login.md]:
```suggestion
- `appSession.secret` - A long, random string
```

**Review by stevehobbsdev — APPROVED** [review]

**Review by pdillon — APPROVED** [review]

**Pull request merged**